pkgs/top-level/static.nix

```diff
@@ -53,6 +53,14 @@ self: super: let
 
 in {
   stdenv = foldl (flip id) super.stdenv staticAdapters;
+  gcc49Stdenv = foldl (flip id) super.gcc49Stdenv staticAdapters;
```

Comment:
I'll have to test this, but I had hoped that these would get the static flags as well. They should all just extend the ```super.stdenv``` attr, but their may be some issue with. There might be some late binding that needs to happen though.

Reply:
I expected that as well, but after adding the first patch the packages from my overlay didn't pick up the static attribute. I had to add these to get a correct build.